phpBB/adm/style/admin.js

```diff
@@ -1,31 +1,29 @@
+/* global phpbb */
+
+/* eslint-disable camelcase, no-unused-vars */
+
 /**
-* phpBB3 ACP functions
+* PhpBB3 ACP functions
```

Comment:
Why capital?

Reply:
it was auto fixed that way as its a js rule not to begin a comment with a lower case letter